pkg/services/ngalert/provisioning/validate.go

```diff
@@ -0,0 +1,5 @@
+package provisioning
+
+import "fmt"
+
+var ErrValidation = fmt.Errorf("invalid route specification")
```

Comment:
As this would be `provisioning.ErrValidation` we might give it a more specific name like `provisioning.ErrRouteValidation` or something like that, wdyt?

Reply:
I tried to keep it generic since we have like 6 resources to provision - I didn't want to need to also create `ErrContactPointValidation` and `ErrAlertRuleValidation` and...

But, I'm also realizing that the text is not generic, so I'll fix that.